staging/src/k8s.io/apiserver/pkg/endpoints/request/webhook_duration.go

```diff
@@ -96,6 +115,19 @@ type LatencyTrackers struct {
 	// ValidatingWebhookTracker tracks the latency incurred in validating webhook(s).
 	// Validate webhooks are done in parallel, so max function is used.
 	ValidatingWebhookTracker DurationTracker
+
+	// StorageTracker tracks the latency incurred inside the storage layer.
+	StorageTracker DurationTracker
+
+	// TransformTracker tracks the latency incurred in transforming the
```

Comment:
This is transforming the response object into the requested groupversion including only the cost of conversion or also the cost of serializing to JSON or proto?

Reply:
it does not include serialization or writing of the serialized bytes to http ResponseWriter object, I made the comments more clear

Reply:
I added serialization tracker, so we measure transform, serialize, writing to the http ResponseWrite independently